### PR TITLE
add a force re-registration option to the scan script

### PR DESCRIPTION
**Pull request by atomic111**

### CODE REVIEW

**Comment by atomic111** [comment]
@cpuschma please have a look and provide me some feedback

File: mdm-scripts/windows/scan.ps1 [hunk 1]
```diff
@@ -11,6 +11,8 @@
     Set 'cnspec' (default) to download from a share and execute it
     .PARAMETER RegistrationToken
     Is required to register the Mondoo Product, if ConfigFile is not existent
+    .PARAMETER ForceRegistration
+    Is required to force re-registration the Mondoo Product and overwrite the existing ConfigFile
```
Comment (cpuschma) [inline mdm-scripts/windows/scan.ps1:15]: I would rephrase the description to explicitly state that the registered client will be logged out and re-registered

**Review by cpuschma — APPROVED** [review]
@atomic111 LGTM
